packages/grafana-schema/src/common/mudball.cue

```diff
@@ -198,7 +198,7 @@ PercentChangeColorMode: "standard" | "inverted" | "same_as_value" @cuetsy(kind="
 FieldTextAlignment: "auto" | "left" | "right" | "center" @cuetsy(kind="type")
 
 // Controls the value alignment in the TimelineChart component
-TimelineValueAlignment: "center" | "left" | "right" @cuetsy(kind="type")
```

Comment:
This is probably a breaking change that Levitate picks up, this changes `TimelineValueAlignment` from a `Type` in typescript to an `Enum`. Is this is a necessary change?